src/components/ha-sidebar.ts

```diff
@@ -980,7 +980,7 @@ class HaSidebar extends LitElement {
           line-height: 20px;
           text-align: center;
           padding: 0px 6px;
-          color: var(--text-accent-color, var(--text-primary-color));
+          color: var(--text-accent-color, var(--sidebar-background-color));
```

Comment:
This would not be the correct solution, we have `--text-accent-color`, meaning the color text should be on an accent color background (the notification badge). That should be set if we want to change the text color on the notification badge.

Reply:
My reasoning for this solution is that the sidebar background contrasts with the notification badge/bell background, hence the text that is put on the notification badge would also contrast.  This is true for both light and dark default themes.

Or is the objection is that a theme variable shouldn't be used for background in some area of the UI and text in another?  If so, I guess that be fixed in other areas of the UI as well?